hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1007 @@
+    "curr_time_mtptg = Time(time_data.mjd, format=\"mjd\", scale=\"tai\")\n",
+    "time_err = curr_time_mtptg - Time.now()\n",
+    "print(f\"Time error={time_err.sec:0.2f} sec\")\n",
+    "\n",
+    "# Compute RA/Dec for commanded az/el\n",
+    "cmd_elaz = AltAz(alt=alt, az=az, \n",
+    "                 obstime=curr_time_mtptg.tai, \n",
+    "                 location=location)\n",
+    "cmd_radec = cmd_elaz.transform_to(ICRS)\n",
+    "lst = Angle(time_data.lst, unit=u.hour)\n",
+    "\n",
+    "para_ang=parallactic_angle(location, lst, cmd_radec).to(u.deg)\n",
+    "print(f\"Alt: {alt}\\nAz: {az}\\nRotTel: {rot_tel}\\nParaAng: {para_ang}\")\n",
+    "\n",
+    "\n",
+    "# Calculating the other parameters     \n",
+    "rot_pa = para_ang-180*u.deg+rot_tel\n",
+    "print(rot_pa)\n",
+    "\n",
+    "await rot.cmd_stop.start(timeout=10.)\n",
+    "await asyncio.sleep(5.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"start tracking\")\n",
+    "await rot.cmd_trackStart.start(timeout=30.)\n",
+    "    \n",
+    "ack = await mtptg.cmd_raDecTarget.set_start(\n",
+    "    targetName=target_name,\n",
+    "    frame=MTPtg.CoordFrame.ICRS,\n",
+    "    epoch=2000,  # should be ignored: no parallax or proper motion\n",
+    "    equinox=2000,  # should be ignored for ICRS\n",
+    "    ra=cmd_radec.ra.hour,\n",
+    "    declination=cmd_radec.dec.deg,\n",
+    "    parallax=0,\n",
+    "    pmRA=0,\n",
+    "    pmDec=0,\n",
+    "    rv=0,\n",
+    "    dRA=0,\n",
+    "    dDec=0,\n",
+    "    trackId=9999,\n",
+    "    rotAngle=15.0,\n",
+    "    rotStartFrame=MTPtg.RotFrame.FIXED,\n",
+    "    rotTrackFrame=MTPtg.RotFrame.FIXED,\n",
+    "    rotMode=MTPtg.RotMode.FIELD,\n",
+    "    azWrapStrategy=2,\n",
+    "    timeOnTarget=30,\n",
+    "    timeout=10\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtptg.cmd_stopTracking.start(timeout=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Start Test"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Test Start\")\n",
+    "\n",
+    "now = datetime.now()\n",
+    "script.evt_logMessage.set_put(level=logging.INFO+1,\n",
+    "                              message=f\"START - {test_message} - Positive Interlock Test - {now} UTC\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set Low Velocity and Move Through Positive Interlock"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#The velocitySet command gets accepted but the moveConstantVelocity command gets error.\n",
+    "#moveConstantVelocity has not been implemented\n",
```

Comment:
MTRotator has no velocitySet or moveConstantVelocity command in the XML. It might have had those at one time. (Use tracking to move at constant velocity). I suggest removing these cells. Testing something that isn't in the API seems a bit nuts.

Reply:
velocitySet and moveConstantVelocity  are taken out. Warning added to revise way to set velocities before using next time the notebook